README.md

```diff
@@ -1,37 +1,29 @@
-# 自定义物料初始模板
+# CRA Materials
 
-## 简介
+Materials for Create React App. Make CRA use in iceworks.
 
-用于自定义物料，支持自定义区块、布局、模板
+## Install
 
-## 使用
+Install iceworks CLI tool:
 
-#### 安装依赖
-
-```
-$ npm install
+```bash
+$ npm i -g iceworks
 ```
 
-#### 区块预览
+## Usage
 
-```
-$ cd blocks/ExampleBlock
-$ npm install
-$ npm run start
+Start iceworks:
+
+```bash
+$ iceworks  # will open http://127.0.0.1:8000 in browser
 ```
 
-#### 模板预览
+Add CRA materials, go to Materials Tab and click Add Button:
 
-```
-$ cd scaffolds/ExampleScaffold
-$ npm install
-$ npm run start
-```
+![](https://img.alicdn.com/tfs/TB1o5j2bYr1gK0jSZR0XXbP8XXa-2872-1634.jpg)
 
-## 添加物料
+Fill out the form. CRA materials url `http://ice.alicdn.com/assets/materials/cra-materials.json`
```

Comment:
Fill ~~out~~ up the form. CRA materials **source** url `http://ice.alicdn.com/assets/materials/cra-materials.json`